components/logging.js

```diff
@@ -129,9 +132,14 @@ async function fetchLogsFromArchive(
     params: {
       "time_window.start_time.seconds": startTimeSeconds,
       "time_window.end_time.seconds": endTimeSeconds,
+      page_size: 50,
```

Comment:
This can be 100 for delivery vehicles

Reply:
Yup looks like it's 300 for trips, but for delivery vehicles, I get an error that says "Maximum number of results per page is 100" and for non-delivery vehicles I get the "max 50" limit error. Should we file a case to ask for the server to not error out since the comments at https://source.corp.google.com/piper///depot/google3/google/maps/fleetengine/v1/archive_api.proto;rcl=464007599;l=417 say that

// The maximum number of trips to return. The service may return fewer than
// this value.

Reply:
Yea that's correct. It's 50 for vehicles, 300 for trips/tasks/, and 100 for delivery vehicles. This is due to size limits when returning results from BigQuery. Go ahead and file a bug for that under Fleet Archive :)